uninstall.sh

```diff
@@ -1,7 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 # uninstall by (github: bernardofire)
 # Remove line from bashrc and zshrc if present.
-for rc in bashrc zshrc; do
-  sed -i '/scm_breeze/d' "$HOME/.$rc"
-  printf "Removed SCM Breeze from %s\n" "$HOME/.$rc"
-done
+
+if [[ $OSTYPE == "Darwin" ]]; then 
+  for rc in bashrc zshrc; do
+    sed -i '' '/scm_breeze/d' "$HOME/.$rc"
+    printf "Removed SCM Breeze from %s\n" "$HOME/.$rc"
+  endfor
```

Comment:
Don't think you meant to leave this in? That isn't valid sh that I know about. =D

Reply:
+1, the for loop syntax in bash is `for ... do ... done`